BubbleSortMainClass.java

```diff
@@ -0,0 +1,154 @@
+
+**
```

Comment:
Can you move the main method to the other file, and delete this file? I'd like to keep BubbleSort as the original name.